schemas/plugins/optimize/aggressiveSplittingPluginSchema.json

```diff
@@ -0,0 +1,23 @@
+{
+	"id": "AggressiveSplittingPlugin",
```

Comment:
Please use two spaces/one tab for indent

Reply:
If this is just github displaying tabs `\t` weirdly forget what I said then 😛

Reply:
I think this is just a github issue 😸 

![image](https://user-images.githubusercontent.com/5036933/32646409-bc26b7ce-c5ec-11e7-95dc-0f7a94d3937a.png)